docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,170 @@
+- [Interop Marshaling](../framework/interop/interop-marshaling.md)
+- [Determine which .NET Framework versions are installed](../framework/migration-guide/how-to-determine-which-versions-are-installed.md)
+- [How to: Configure Network Tracing](../framework/network-programming/how-to-configure-network-tracing.md)
+- [How to: customize a time-based cache policy](../framework/network-programming/how-to-customize-a-time-based-cache-policy.md)
+- [Using TCP Services](../framework/network-programming/using-tcp-services.md)
+- [Application Domain Resource Monitoring (ARM) ETW Events](../framework/performance/application-domain-resource-monitoring-arm-etw-events.md)
+- [Garbage Collection ETW Events](../framework/performance/garbage-collection-etw-events.md)
+- [Creating Satellite Assemblies for Desktop Apps](../framework/resources/creating-satellite-assemblies-for-desktop-apps.md)
+- [.NET Framework Tools](../framework/tools/index.md)
+- [How to: Import Custom WSDL](../framework/wcf/extending/how-to-import-custom-wsdl.md)
+- [How to: Write an Extension for the ServiceContractGenerator](../framework/wcf/extending/how-to-write-an-extension-for-the-servicecontractgenerator.md)
+- [Flowing Transactions into and out of Workflow Services](../framework/wcf/feature-details/flowing-transactions-into-and-out-of-workflow-services.md)
+- [Custom WSDL Publication](../framework/wcf/samples/custom-wsdl-publication.md)
+- [Extending Control Over Error Handling and Reporting](../framework/wcf/samples/extending-control-over-error-handling-and-reporting.md)
+- [How to create and run a long-running workflow](../framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md)
+- [About GDI+ Managed Code](../framework/winforms/advanced/about-gdi-managed-code.md)
+- [How to: Read Image Metadata](../framework/winforms/advanced/how-to-read-image-metadata.md)
+- [Threading Model](../framework/wpf/advanced/threading-model.md)
+- [Navigation Overview](../framework/wpf/app-development/navigation-overview.md)
+- [Adorners Overview](../framework/wpf/controls/adorners-overview.md)
+- [Data Templating Overview](../framework/wpf/data/data-templating-overview.md)
+- [How to: Control When the TextBox Text Updates the Source](../framework/wpf/data/how-to-control-when-the-textbox-text-updates-the-source.md)
+
+## F# language
+
+- [Get Started with F# in Visual Studio Code](../fsharp/get-started/get-started-vscode.md)
+- [F# docs - get started, tutorials, reference.](../fsharp/index.yml)
+- [Computation Expressions](../fsharp/language-reference/computation-expressions.md)
+- [Sequences](../fsharp/language-reference/sequences.md)
+- [F# code formatting guidelines](../fsharp/style-guide/formatting.md)
+- [What's new in F# 4.5 - F# Guide](../fsharp/whats-new/fsharp-45.md)
+- [What's new in F# 4.6 - F# Guide](../fsharp/whats-new/fsharp-46.md)
+- [What's new in F# 4.7 - F# Guide](../fsharp/whats-new/fsharp-47.md)
+
+## Contributors
```

Comment:
Make it clear it's only external so FTEs don't complain they're not being listed

```suggestion
## External contributors
```

Reply:
I chose "Community contributors" instead of "External". What do you think?